pkg/agent/plugin/workloadattestor/v0_test.go

```diff
@@ -0,0 +1,68 @@
+package workloadattestor_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/spiffe/spire/pkg/agent/plugin/workloadattestor"
+	"github.com/spiffe/spire/pkg/common/catalog"
+	workloadattestorv0 "github.com/spiffe/spire/proto/spire/agent/workloadattestor/v0"
+	"github.com/spiffe/spire/proto/spire/common"
+	"github.com/spiffe/spire/test/spiretest"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestV0(t *testing.T) {
+	expected := map[int][]*common.Selector{
+		1: {},
+		2: {{Type: "not", Value: "relevant"}},
+	}
+
+	t.Run("attest fails", func(t *testing.T) {
+		workloadAttestor := makeFakeV0Plugin(t, expected)
+		_, err := workloadAttestor.Attest(context.Background(), 0)
+		spiretest.RequireGRPCStatus(t, err, codes.InvalidArgument, "workloadattestor(test): ohno")
+	})
+
+	t.Run("no selectors for pid", func(t *testing.T) {
+		workloadAttestor := makeFakeV0Plugin(t, expected)
+		actual, err := workloadAttestor.Attest(context.Background(), 1)
+		require.NoError(t, err)
+		require.Empty(t, actual)
+	})
+
+	t.Run("no selectors for pid", func(t *testing.T) {
```

Comment:
Looks like this should be "with selectors..."?

Reply:
:facepalm: thanks!